index.html

```diff
@@ -1012,6 +1012,92 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `file_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest">`file_handlers`</dfn></code> member is a [=list=] that
+          represents urls within the app that can handle launches of a given
+          file type (TODO: link to definition of file type).
+        </p>
+        <p>
+          How file handlers are registered and disambiguated are at the
+          discretion of the operating system.
+        </p>
+        <p>
+          To <dfn>process the `file_handlers` member</dfn>, given [=ordered
+          map=] |json:ordered map|, [=ordered map=] |manifest:ordered map|:
+        </p>
+        <ol class="algorithm">
+          <li>Let |processedFileHandlers:list| be a new [=list=].
+          </li>
+          <li>Set |manifest|["file_handlers"] to |processedFileHandlers|.
+          </li>
+          <li>If |json|["file_handlers"] doesn't [=map/exist=] or
+          |json|["file_handlers"] is not a [=list=], return.
+          </li>
+          <li>[=list/For each=] |entry:ordered map| of |json|["file_handlers"]:
+            <ol>
+              <li>Let |file_handler:ordered map| be [=process a file handler
+              item=] with |entry|.
+              </li>
+              <li>If |file_handler| is failure, continue.
+              </li>
+              <li>[=list/Append=] |file_handler| to |processedFileHandlers|.
+              </li>
+            </ol>
+          </li>
+        </ol>
+        <p>
+          A user agent SHOULD register the file handlers with the operating
+          system via interactions that are consistent with:
+        </p>
+        <ul>
+          <li>The [=file handler/icon=] and [=file handler/name=] appearing in
+          an "Open with.." menu in the context menu of associated files.
```

Comment:
```suggestion
          an "Open with…" menu in the context menu of associated files.
```

Reply:
FWIW, GitHub is rendering @tomayac's suggestion as a skinny ellipsis character (`…`) rather than just three periods (`...`). I'm not sure which was intended.

Reply:
... but be consistent in the doc.